http/server.go

```diff
@@ -80,10 +82,11 @@ func (server *RESTServer) CompleteRequest(start time.Time, context *Context) {
 		}(start)
 	}
 	contentType := context.Response.Header().Get(contentTypeHeader)
-	if stringutil.IsWhiteSpace(context.Response.Header().Get(contentTypeHeader)) {
+	if stringutil.IsEmpty(contentType) {
```

Comment:
Why the switch from `IsWhiteSpace` to `IsEmpty ` ?

Reply:
Unintentional